src/static/courses.ts

```diff
@@ -135,11 +133,25 @@ export const coursesInfo = [
           'In this interactive project, you will be guided through building and deploying a smart contract to the NEAR testnet blockchain.'
         ]
       },
+      {
+        title: 'Learn NEAR Smart Contracts by Building a Word Guessing Game',
+        type: projectTypes.practice,
+        summary: [
+          'In this interactive project, you will learn more features of NEAR smart contracts while building a word guessing game.'
+        ]
+      },
+      {
+        title: 'Build a Sentence Making Smart Contract',
```

Comment:
I think this should be the correct name.

```suggestion
        title: 'Build a Sentence-Making Smart Contract',
```

Reply:
thanks, but too late 😬